scheduler/src/cook/mesos/mesos_compute_cluster.clj

```diff
@@ -26,6 +26,7 @@
             [cook.plugins.pool :as pool-plugin]
             [cook.progress :as progress]
             [cook.scheduler.scheduler :as sched]
+            [cook.tools :as util]
```

Comment:
Import this as tools, not util. Should be easy search&replace.

Reply:
Done